public/app/features/alerting/unified/components/rules/RuleStats.tsx

```diff
@@ -41,75 +42,65 @@ export const RuleStats: FC<Props> = ({ showInactive, showRecording, group, names
       }
       stats.total += 1;
     };
+
     if (group) {
       group.rules.forEach(calcRule);
     }
+
     if (namespaces) {
       namespaces.forEach((namespace) => namespace.groups.forEach((group) => group.rules.forEach(calcRule)));
     }
+
     return stats;
   }, [group, namespaces]);
 
   const statsComponents: React.ReactNode[] = [];
+
   if (calculated[PromAlertingRuleState.Firing]) {
     statsComponents.push(
-      <StateColoredText key="firing" status={PromAlertingRuleState.Firing}>
-        {calculated[PromAlertingRuleState.Firing]} firing
-      </StateColoredText>
+      <Badge color="red" key="firing" text={`${calculated[PromAlertingRuleState.Firing]} firing`} />
     );
   }
+
   if (calculated.error) {
-    statsComponents.push(
-      <StateColoredText key="errors" status={PromAlertingRuleState.Firing}>
-        {calculated.error} errors
-      </StateColoredText>
-    );
+    statsComponents.push(<Badge color="red" key="errors" text={`${calculated.error} errors`} />);
   }
+
   if (calculated[PromAlertingRuleState.Pending]) {
     statsComponents.push(
-      <StateColoredText key="pending" status={PromAlertingRuleState.Pending}>
-        {calculated[PromAlertingRuleState.Pending]} pending
-      </StateColoredText>
+      <Badge color={'orange'} key="pending" text={`${calculated[PromAlertingRuleState.Pending]} pending`} />
     );
   }
-  if (showInactive && calculated[PromAlertingRuleState.Inactive]) {
+
+  if (calculated[PromAlertingRuleState.Inactive]) {
     statsComponents.push(
-      <StateColoredText key="inactive" status="neutral">
-        {calculated[PromAlertingRuleState.Inactive]} normal
-      </StateColoredText>
+      <Badge color="green" key="inactive" text={`${calculated[PromAlertingRuleState.Inactive]} normal`} />
```

Comment:
I wonder if we should display a badge for `Inactive/Normal` state. It doesn't exist for Mimir/Prom rules so the behaviour will be a bit inconsistent here

Reply:
I believe ease of use is more important than consistency as long as we keep track of what we could land upstream to eventually reach consistency.